perfkitbenchmarker/linux_benchmarks/mnist_benchmark.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Run MNIST benchmarks."""
+
+import re
+from perfkitbenchmarker import configs
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import sample
+from perfkitbenchmarker.providers.gcp import gcp_tpu
+
+FLAGS = flags.FLAGS
+
+BENCHMARK_NAME = 'mnist'
+BENCHMARK_CONFIG = """
+mnist:
+  description: Runs MNIST Benchmark.
+  vm_groups:
+    default:
+      vm_spec:
+        GCP:
+          image: debian-9-tf-v20171002
```

Comment:
does this require debian?

Reply:
Also, is there anyway we can be consistent and install tensorflow from source, as we do in the other tensorflow benchmarks?

Reply:
I want to use preinstall tensorflow images from ml-images. And currently, all the tf-nightly images is debian images

Reply:
I want to use preinstalled tensorflow image from ml-images. Now all the tf-nightly images is debian images in gcp

Reply:
Alright, seems fine since this benchmark can only run on GCP.